ros_buildfarm/templates/release/release_reconfigure-jobs_job.xml.em

```diff
@@ -13,6 +13,16 @@
     priority=30,
 ))@
 @(SNIPPET(
+    'property_parameters-definition',
+    parameters=[
+        {
+            'type': 'boolean',
+            'name': 'dry_run',
```

Comment:
since this will become an environment variable in the test it would be cleaner to be all caps